tests/nodeos_startup_catchup.py

```diff
@@ -102,23 +108,34 @@ def waitForNodeStarted(node):
             sleepTime+=1
 
     node0=cluster.getNode(0)
+    producerP2pPort = cluster.getNodeP2pPort(0)
 
     Print("Wait for account creation to be irreversible")
     blockNum=head(node0)
     waitForBlock(node0, blockNum, blockType=BlockType.lib)
 
-    Print("Startup txn generation")
-    period=1500
-    transPerPeriod=150
-    for genNum in range(0, len(txnGenNodes)):
-        salt="%d" % genNum
-        txnGenNodes[genNum].txnGenStart(salt, period, transPerPeriod)
-        time.sleep(1)
+    Print("Configure txn generators")
+    info = node0.getInfo()
+    chainId = info['chain_id']
+    lib_id = info['last_irreversible_block_id']
+
+    targetTpsPerGenerator = 100
+    targetTps = targetTpsPerGenerator*trxGeneratorCnt
+    tpsLimitPerGenerator=targetTpsPerGenerator
+    testTrxGenDurationSec=60*60
+
+    tpsTrxGensConfig = TpsTrxGensConfig(targetTps=targetTps, tpsLimitPerGenerator=tpsLimitPerGenerator)
+    trxGenLauncher = TransactionGeneratorsLauncher(chainId=chainId, lastIrreversibleBlockId=lib_id,
+                                                   handlerAcct=cluster.eosioAccount.name, accts=f"{account1Name},{account2Name}",
+                                                   privateKeys=f"{account1PrivKey},{account2PrivKey}", trxGenDurationSec=testTrxGenDurationSec,
+                                                   logDir=Utils.DataDir, peerEndpoint=node0.host, port=producerP2pPort, tpsTrxGensConfig=tpsTrxGensConfig)
+
+    Print("Launch txn generators and start generating/sending transactions")
+    trxGenLauncher.launch(waitToComplete=False)
```

Comment:
It looks like this boiler-plate code is pretty much the same except for a few params. Can it all be wrapped into a `launchTrxGenerator` method of `Cluster` or `Node`?

Reply:
I believe this addresses your suggestion: [1842807](https://github.com/AntelopeIO/leap/pull/634/commits/18428070e9bfe91c602965e2253cf733e15c0ecd)